src/openrct2/ride/coaster/BolligerMabillardTrack.hpp

```diff
@@ -39,39 +39,22 @@ static void BolligerMabillardTrackFlat(
 {
     if (trackElement.HasChain())
     {
-        static constexpr const uint32_t imageIds[] = {
```

Comment:
I’m not quite sure about removing this - I thought that @duncanspumpkin wanted to keep arrays or even introduce more sophisticated classes for this.

Reply:
I would like to make a noted document somewhere with all sprites. Especially with notes if their directions aren't in the same system from 0 to 3, which I have sadly come across too often today. But that way I hope all track pieces can be united on the same code. I know that TrackPaint.cpp is supposed to that, but I genuinely cannot make cake of that file. Might have to rewrite it in a system like this. I'll post the ideas to the track type discussion.